src/replication.c

```diff
@@ -1272,6 +1272,10 @@ void syncCommand(client *c) {
  * - rdb-channel <1|0>
  * Used to identify the client as a replica's rdb connection in an dual channel
  * sync session.
+ *
+ * - pseudo-master <0|1>
+ * Set this connection behaving like a master if server.pseudo_replica is true.
+ * Sync tools can set their connections into 'pseudo-master' state to visit expired keys.
  * */
 void replconfCommand(client *c) {
```

Comment:
The sync tool sends `REPLCONF PSEUDO-MASTER` to say "I'm a pseudo-master, so ignore expire for this connection"?

Almost all other REPLCONF commands are sent by the replica to the primary before doing the PSYNC. This one is different. Can we add a more explicit comment about this difference, similar to `REPLCONF GETACK` which is also different:

```
 * - getack <dummy>
 * Unlike other subcommands, this is used by primary to get the replication
 * offset from a replica.
```

Reply:
> The sync tool sends `REPLCONF PSEUDO-MASTER` to say "I'm a pseudo-master, so ignore expire for this connection"?
Yes. It does look a bit strange. I add a new command `CLIENT IMPORT-SOURCE <on|off>` like what @soloestoy say. Maybe it'll look better this way?